package.json

```diff
@@ -6,7 +6,9 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "lint:css": "npx stylelint '**/*.css'",
+    "lint:scss": "npx stylelint '**/*.scss'"
```

Comment:
I'd like to see this set up so that `npm run lint` runs all of our linting scripts in sequence. We can use the `npm-run-all` package to do that.

Here's how I have that set up for my eleventy starter: https://github.com/dustin-jw/eleventy-starter/blob/main/package.json#L16-L26